base/src/main/java/com/twozo/page/company/Company.java

```diff
@@ -0,0 +1,48 @@
+package com.twozo.page.company;
+
+import com.twozo.page.BasePage;
+import com.twozo.page.xpath.XPathBuilder;
+import com.twozo.web.driver.service.WebAutomationDriver;
+import com.twozo.web.element.service.WebPageElement;
+
+import java.util.Objects;
+
+public class Company extends BasePage {
```

Comment:
When you say "company page," are you referring to the list page or the detail page?